docs/QUICKSTART.md

```diff
@@ -3,16 +3,16 @@
 In this tutorial you will:
 
 1. Bootstrap your development environment.
-2. Create, build, and deploy an index to an indexer service hooked up to Fuel's `beta-3` testnet.
-3. Query the indexer service for indexed data using GraphQL.
+2. Create, build, and deploy an indexer to an indexer service hooked up to Fuel's `beta-3` testnet.
```

Comment:
-er it. for ev-er

Reply:
I'm not sure what this means

Reply:
We are forever adding 'er' on to the word index. Too much screen time today for me!